.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: ruby
+rvm:
+  - 2.5.2
```

Comment:
Do we also need postgres in order to run the tests?
https://docs.travis-ci.com/user/database-setup/#postgresql

Reply:
you are right. I will add that :)